crates/wash/src/bin/wash.rs

```
@@ -386,6 +391,8 @@ async fn main() {
         .with_env_filter(EnvFilter::from_default_env())
         .init();
 
+    let _ = inform_new_wash_version().await;
```

```suggestion
    inform_new_wash_version().await?;
```

---

Yeah that's a good point -- I'd prefer if things didn't fail silently, but you're right that it's an optional helpful notice, not required for actually running. 

I think if we go with putting this behind an experimental flag for now we can make sure it passes (then when that experimental flag goes GA, we can allow the failure?)

---

Updated that an error causes the program to terminate.